profiler/lib.profiler/build.xml

```diff
@@ -33,19 +33,19 @@
     <target name="compile-system" depends="init">
         <mkdir dir="${build15.classes.dir}"/>
         <javac srcdir="${src15.dir}" destdir="${build15.classes.dir}" debug="${build.compiler.debug}" debuglevel="${build.compiler.debuglevel}" encoding="UTF-8"
-                deprecation="${build.compiler.deprecation}" optimize="${build.compiler.optimize}" source="1.5" target="1.5" includeantruntime="false">
+                deprecation="${build.compiler.deprecation}" optimize="${build.compiler.optimize}" source="${javac.source}" target="${javac.target}" includeantruntime="false">
```

Comment:
@jlahoda ,please provide your opinion on Matthias concern

Reply:
Good observation @matthiasblaesing. Yes, we want to prevent people to accidentally bump the level of the whole `lib.profiler` module. To keep such check local, I'd suggest to just insert a check on line 35:
```xml
        <fail message="javac.source of lib.profiler classes should be as low as possible!">
            <condition>
                <not>
                    <or>
                        <equals arg1="${javac.source}" arg2="1.5"/>
                        <equals arg1="${javac.source}" arg2="1.7"/>
                    </or>
                </not>
            </condition>
        </fail>
```
that should guarantee people think when they change the `javac.source` and we don't have to make API changes in `common.xml` with unknown consequences.

Btw. the whole PR is still labeled as "don't merge". Is that really necessary? Can't we merge now? I've already approved it.

Reply:
Thanks @JaroslavTulach  I will make this change  and will remove the label "do-not-merge"

Reply:
Had added the check in last commit